dev-libs/boost/boost-1.75.0.ebuild

```diff
@@ -1,14 +1,14 @@
 # Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=6
```

Comment:
From my understanding, you have to pin `MY_PV` to `1_75_0` and `MAJOR_V` to `1_75_0`, because you are using EAPI 6 here.
Is there a specific reason for doing that?
Would it be possible to set EAPI to 7, just like Gentoo does, so that you can use `ver_rs` and `ver_cut` below for `MY_PV` and `MAJOR_V`?

I am asking because in theory we should minimize the amount of code that is specific to Flatcar.
If it is really necessary to customize, then of course we could do so by moving the code to coreos-overlay.
I am just wondering if it is really necessary.

You wrote like `I have some little issue with the library that I think to solve in a short time.` in the [description](https://github.com/kinvolk/Flatcar/issues/362#issue-825432268).
I am not sure exactly what you mean.

Reply:
Hi,
I  switched to EAPI 6 because you use it in your portage version. I did not see any EAPI 7 package in your portage. I can move to _Flatcar Alpha_  and put EAPI to 7 but I need to make test for it.

Reply:
Right, the current ebuilds in Flatcar Alpha, `dev-libs/boost` 1.65.0 and `dev-util/boost-build` 1.65.0, have used EAPI 6.
That's simply because those ebuilds were written a few years ago, when Gentoo EAPI 7 was not widely used.
On the other hand, most ebuilds in the recent days rely on EAPI 7, following Gentoo standard.
For example, please see [app-arch/tar](https://github.com/kinvolk/portage-stable/blob/49516d78506a929c24de0b0546c643a5353367e7/app-arch/tar/tar-1.32.90.ebuild#L4).

As boost and boost-build need to be updated to 1.75, we should also change EAPI to 7 like Gentoo, unless there is a reason for not doing that.

Reply:
With a single test emerge finish the compilation with EAPI 7. I can switch to EAPI 7 if you prefer it but in my opinion it look a bit strange because in your portage directory I did not see any EAPI 7 packages. I only see EAPI=7 in coreos-overlay. 

In my interpretation portage-stable is Gentoo vanilla with minor modification  and with a stability in the system. On the other side coreos-overlay is a Flatcar specific overlay. I noted that you did not use EAPI 7 in portage-stable but you use it in coreos-overlay and I imagined that you did it for the stability of EAPI 7  with the Flatcar portage-stable. If it is not the case I can use EAPI 7 and make the test for the ebuild.

Reply:
Sorry , 
I read now post previous to my last comment (https://github.com/kinvolk/portage-stable/pull/153#discussion_r594262759). Thanks for the explanation , I will move to EAPI 7 and I will make the rebase after all my test round.